Don't remove sounds on dist-clean, upstream tarballs come with
one of them.

 -- Faidon Liambotis <paravoid@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -364,7 +364,6 @@ dist-clean: distclean
 
 distclean: clean
 	@$(MAKE) -C menuselect dist-clean
-	@$(MAKE) -C sounds dist-clean
 	rm -f menuselect.makeopts makeopts menuselect-tree menuselect.makedeps
 	rm -f makeopts.embed_rules
 	rm -f config.log config.status
@@ -374,6 +373,10 @@ distclean: clean
 	rm -rf doc/api
 	rm -f build_tools/menuselect-deps
 
+# tarballs distributed by Digium include sounds
+all-clean: distclean
+	@$(MAKE) -C sounds dist-clean
+
 datafiles: _all
 	if [ x`$(ID) -un` = xroot ]; then CFLAGS="$(ASTCFLAGS)" sh build_tools/mkpkgconfig $(DESTDIR)/usr/lib/pkgconfig; fi
 # Should static HTTP be installed during make samples or even with its own target ala
